### DISCUSSION TITLE
Vue-apollo with nuxt, nuxt-module is not updating

### DISCUSSION THREAD

**Discussion by rnenjoy** [Akryum/vue-apollo#484] [General]
Lately it is very hard to use vue-apollo with nuxt cause the nuxt-module is not updating and works together with the latests versions of vue-apollo. Maybe this author of this great library can take care of doing its own module that works with the latest release? :)

**Comment by lautr** [comment]
I think a lot of people would appreciate that, there isn't really a stable & working release out there yet. And especially support for SSR is an issue, from a documentation & functional point of view.

**Comment by fvigotti** [comment]
my 2c as newbye in this ecosystem : would be also good to have a guide for implementation of vue-apollo with nuxt, because all references out there say to use nuxt-module but it's not usable because outdated/unmaintained
